app/Http/Controllers/FollowController.php

```diff
@@ -2,25 +2,25 @@
 
 namespace App\Http\Controllers;
 
-use App\Repository\FollowRepository;
+use App\Repository\Interfacee\FollowRepositoryI;
```

Comment:
```suggestion
use App\Repository\Interfacee\FollowRepositoryInterface;
```
fix tương tự các chỗ khác nha :3